infra/gcp/namespaces/ensure-namespaces.sh

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env bash
+#
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script creates namespaces, applies roles and role bindings and
+# checks if assigned role binding is the only one in the namespace
+# for k8s-infra projects
```

Comment:
document this assumes kubectl in $PATH

Reply:
I've added the flag `--kubectl-path | -p` where you can specify path for kubectl